roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -175,7 +175,7 @@
     state: started
 
 - name: Wait for Under Replicated Partitions on Broker
-  include_tasks: tasks/wait_for_urp.yml
+  include_tasks: ../../tasks/wait_for_urp.yml
```

Comment:
@JumaX this change was to make the molecule converge command work, i think it will also need to be made for your rbac work